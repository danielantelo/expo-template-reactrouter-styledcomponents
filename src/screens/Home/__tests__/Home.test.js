import React from 'react';
import { create } from 'react-test-renderer';
import HomeScreen from '..';

describe('HomeScreen', () => {
  const renderer = create(<HomeScreen />);

  it('renders', () => {
    expect(renderer.toJSON()).toMatchSnapshot();
  });

  it('has welcome text', () => {
    renderer.root.findByProps({ children: 'Welcome Home!' });
  });
});
