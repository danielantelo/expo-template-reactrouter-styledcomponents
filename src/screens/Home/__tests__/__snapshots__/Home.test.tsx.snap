// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen renders 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RCTScrollView>
    <View>
      <Text
        className="sc-AxiKw ibkFZR"
      >
        Welcome Home!
      </Text>
      <View
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={null}
      >
        <Text
          style={null}
        >
          Go to Settings
        </Text>
      </View>
    </View>
  </RCTScrollView>
  <View>
    <View
      className="sc-AxjAm eEzWIB"
    >
      <View
        className="sc-AxirZ eEmRVZ"
      >
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={null}
        >
          <Text
            style={null}
          >
            Home
          </Text>
        </View>
      </View>
      <View
        className="sc-AxirZ eEmRVZ"
      >
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={null}
        >
          <Text
            style={null}
          >
            Settings
          </Text>
        </View>
      </View>
    </View>
  </View>
</RCTSafeAreaView>
`;
